pkg/cmd/cli/backstage/rest.go

```diff
@@ -7,9 +7,7 @@ import (
 	"github.com/go-resty/resty/v2"
 	"github.com/redhat-ai-dev/model-catalog-bridge/pkg/config"
 	"github.com/redhat-ai-dev/model-catalog-bridge/pkg/rest"
-	"github.com/redhat-ai-dev/model-catalog-bridge/test/stub/common"
```

Comment:
losing this import in rest.go was the key bit @johnmcollier for getting the compile to work from the Dockerfile / docker build / podman build